roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -29,6 +29,17 @@
   failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
   changed_when: osmiq_create_service_account.rc == 0
 
+- name: Create Image Inspector Service Account
+  shell: >
+    echo {{ manageiq_image_inspector_service_account | to_json | quote }} |
+    {{ openshift.common.client_binary }} create
+    -n management-infra
+    --config={{manage_iq_tmp_conf}}
+    -f -
+  register: osmiq_create_service_account
+  failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
+  changed_when: osmiq_create_service_account.rc == 0
```

Comment:
I'm not going to ask for you to refactor this now, since we are already doing this in multiple places, just wanted to add a comment and get feedback from @abutcher and @sdodson on trying to move away from this pattern and separating these into two separate tasks, one that checks if the account exists and another that attempts to create the account if it doesn't exist.

Thoughts?